extensions/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/NotFoundExceptionMapper.java

```diff
@@ -215,6 +216,11 @@ public Response toResponse(NotFoundException exception) {
                 .fromBoundResourceInvokers(bounded
                         .entrySet());
 
+        descriptions.sort(RESOURCE_DESCRIPTION_COMPARATOR);
+        for (ResourceDescription description : descriptions) {
+            description.calls.sort(METHOD_DESCRIPTION_COMPARATOR);
+        }
```

Comment:
I think I would have moved this to `fromBoundResourceInvokers` but it's a detail.

Can you take care of it, squash everything and put this out of draft?

Reply:
Okay, I moved it into fromBoundResourceInvokers.